pkg/common/constants.go

```diff
@@ -44,4 +44,11 @@ const (
 	ContexLocalSsdCacheSize = "local-ssd-cache-size"
 	// Node name for E2E tests
 	TestNode = "test-node-csi-e2e"
+
+	// Default LSSD count for datacache E2E tests
+	LocalSSDCountForDataCache = 2
+
+	// Node label for datacache
```

Comment:
Add a comment saying "these labels are only available on GKE" because we have other customers deploy PDCSI driver on GCE.